src/tape/ensemble.py

```diff
@@ -1021,6 +1021,9 @@ def batch(self, func, *args, meta=None, use_map=True, compute=True, on=None, **k
             Designates which column(s) to groupby. Columns may be from the
             source or object tables. For TAPE and `light-curve` functions
             this is populated automatically.
+        label: 'str', optional
+            If provided the ensemble will use this label to track the result 
+            dataframe. If `None`, the frame will not be tracked. 
```

Comment:
Having the default be that the frame isn't tracked feels not ideal. Users may not set the name out of ignorance and then realize their result is not within the ensemble. Could we have this generate a dummy name by default "result1" that increments? And None disentangles it from the Ensemble as described above.

Reply:
Done.